pkgs/tools/networking/termshark/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "termshark-${version}";
```

If `name` is not set and `pname` is set, then set `name` to `${pname}-${version}`.

```suggestion
  pname = "termshark";
```